src/compat.h

```diff
@@ -578,6 +718,19 @@ get_attname_compat(Oid relid, AttrNumber attnum, bool missing_ok)
 #define PreventInTransactionBlock PreventTransactionChain
 #endif
 
+/*
+ * table_beginscan
+ * PG12 generalizes table scans to those no directly dependant on the heap.
```

Comment:
```suggestion
 * PG12 generalizes table scans to those not directly dependent on the heap.
```

Not sure I understand this formulation either.